README.md

```diff
@@ -152,7 +152,28 @@ Make copies of the [example manifests](./manifests/example) for editing. The nec
 
 ### Cloning VM Images
 
-Cloning is achieved by creating a new PVC with the 'k8s.io/CloneRequest' annotation indicating the name of the PVC the image is copied from. Once the controller detects the PVC, it starts two pods which are responsible for the cloning of the image from one PVC to another using a unix socket that is created on the host itself. When the cloning is completed, the PVC which the image was copied to, is assigned with the 'k8s.io/CloneOf' annotation to indicate cloning completion. The copied VM image can be used by a new pod only after the cloning process is completed.
+Cloning is achieved by creating a new PVC with the 'k8s.io/CloneRequest' annotation indicating the name of the PVC the image is copied from.
+Once the controller detects the PVC, it starts two pods which are responsible for the cloning of the image from one PVC to another using a unix socket that is created on the host itself.
+When the cloning is completed, the PVC which the image was copied to, is assigned with the 'k8s.io/CloneOf' annotation to indicate cloning completion.
+The copied VM image can be used by a new pod only after the cloning process is completed.
+
+Cloning requires two pods, the source pod and target pod, which must execute on the same node.
+Pod adffinity is used to enforce this requirement; however, the cluster also needs to be configured to delay volume binding until pod scheduling has completed.
+
+In Kubernetes 1.10 and older export KUBE_FEATURE_GATES before bringing up the cluster:
```

Comment:
Not required in 1.11?

Reply:
That's my understanding. @aglitke ?

Reply:
I thought the following feature gates are defaulted to true starting in 1.10 (promoted to beta)...so only needed in 1.9 and below...I could be mistaken?

Reply:
Correct @screeley44 . I've updated the readme. Thanks!